utest/shared/src/main/scala/utest/Tests.scala

```diff
@@ -114,7 +114,7 @@ object Tests{
           ..$normal2
           ${
           if (childCallTrees.isEmpty) q"_root_.scala.Left($last)"
-          else q"$last; _root_.scala.Right(Array(..$childCallTrees))"
```

Comment:
I just hit an issue where I have a local type `Array` in scope at a `Tests {}` call-site, so this needed to be `_root_`-prefixed, but happy to see it was already fixed for a different reason 🎉 thanks!